tiledb/common/test/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ commence(unit_test memory_tracker)
 conclude(unit_test)
 
 commence(unit_test common_utils)
-    this_target_sources(main.cc unit_alt_var_length_view.cc unit_iterator_facade.cc unit_permutation_sort.cc unit_permutation_view.cc unit_proxy_sort.cc unit_sort_zip.cc unit_var_length_util.cc unit_var_length_view.cc unit_zip_view.cc)
+    this_target_sources(main.cc ../util/test/unit_alt_var_length_view.cc ../util/test/unit_iterator_facade.cc ../util/test/unit_permutation_sort.cc ../util/test/unit_permutation_view.cc ../util/test/unit_proxy_sort.cc ../util/test/unit_sort_zip.cc ../util/test/unit_var_length_util.cc ../util/test/unit_var_length_view.cc ../util/test/unit_zip_view.cc)
```

Comment:
This `unit_test` environment no longer belongs in `common/test`. It belongs in `common/util/test`.

The principle is that we want unit tests to be defined with the greatest locality possible. There should never need to be a directory specification in a `unit_test` environment.